convert2rhel/rhelvariant.py

```diff
@@ -33,6 +40,17 @@ def determine_rhel_variant():
     if not tool_opts.variant:
         tool_opts.variant = _user_to_choose_rhel_variant()
     loggerinst.info("Variant: %s" % tool_opts.variant)
+
+    # checking RHN Classic
```

Comment:
Actually, the detection of RHN Classic has nothing to do with determining RHEL variant. Please move the code from `rhelvariant.py` to `subscription.py`. Add this piece of code to a separate function, like `unregister_from_rhn_classic`. And call that from `main.py`, for example right before calling `pkghandler.remove_blacklisted_pkgs()`.

Reply:
> And call that from main.py, for example right before calling pkghandler.remove_blacklisted_pkgs().

Not resolved ^